src/map/zone_entities.h

```diff
@@ -71,6 +72,7 @@ class CZoneEntities
     bool			CharListEmpty();
     uint16			GetNewTargID();
 
+    EntityList_t	m_allyList;
```

Comment:
probably better to define this in CInstance - it has no relevance to a regular zone (and just adds confusion with the bcnms ally list vs the zones, if it's visible)